tmt/config/models/hardware.py

```diff
@@ -0,0 +1,13 @@
+from typing import Optional
+
+from tmt.container import MetadataContainer
+
+
+class Translate(MetadataContainer):
+    name: str
+    child_name: Optional[str]
+    element: str
+
+
+class HardwareConfig(MetadataContainer):
+    hardware_config: list[Translate]
```

Comment:
I'd probably call it "translations" or something. `Config.hardware.hardware_config` is really puzzling.